src/rf/apps/workers/process.py

```diff
@@ -58,48 +63,84 @@ def start_polling(self):
                         delete_message = self.check_timeout(record['data'])
                     elif job_type in JOB_VALIDATE:
                         if record['eventSource'] == 'aws:s3':
-                            delete_message = self.validate_image(record)
+                            delete_message = self.__validate_image(record)
 
                 if delete_message:
                     self.q.delete_message(message)
 
-    def validate_image(self, data):
+    def __validate_image(self, data):
```

Comment:
I'm planning to make them all private but wanted to keep the changes scoped to the actual work being done here. As we add to the other jobs I intended to make them private as well.